packages/0xcert-ethereum-erc721-contracts/src/contracts/nf-token-metadata.sol

```diff
@@ -114,7 +119,8 @@ contract NFTokenMetadata is
 
   /**
    * @dev Contract constructor.
-   * @notice When implementing this contract don't forget to set nftName, nftSymbol and uriBase.
+   * @notice When implementing this contract don't forget to set nftName, nftSymbol, uriPrefix and
```

Comment:
* @notice When implementing this contract, don't forget to set nftName, nftSymbol, uriPrefix and